### PR TITLE
Add provenance signing

### PR DESCRIPTION
**Pull request by billnapier**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+github-webhook-pubsub-bridge
```
Comment (google-admin) [inline .gitignore:1]: Was this intended to be in this PR?

Comment (Laptop765) [inline .gitignore:1]: Was this intended to be in this PR?

Comment (Laptop765) [inline .gitignore:1]: Was this intended to be in this PR?

**Review by Laptop765 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (billnapier) [reply .gitignore:1]: That has my intention. But it can also go in a different PR.

This PR is already a dumpster fire....

**Pull request merged**